grunt/service-worker.tmpl

```diff
@@ -133,9 +133,7 @@ self.addEventListener('fetch', function(event) {
     // But it fails when running e2e test
     // this has been attributed by the root url serving both json GET request and the cached HTML page
     if (event.request.headers.get('accept').indexOf('application/json') !== -1) {
-      return event.respondWith(async function() {
-        return fetch(event.request);
-      }());
+      return event.respondWith(fetch(event.request));
```

Comment:
I _think_ this is equivalent... it seems to work.

Reply:
:P lol